client/src/index.js

```diff
@@ -26,6 +29,8 @@ render(
       <Route path="/" element={<App />}></Route>
       <Route path="/signin" element={<SignIn />}></Route>
       <Route path="/signup" element={<SignUp />}></Route>
+      <Route path="/student" element={<Student />}></Route>
```

Comment:
remove this, I deleted the student page because its unnecessary

Reply:
Deleted and Resolved